1.Normalize/scripts/nbconverted/1.Cell-Data-Normalization.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[20]:
+
+
+import pycytominer
+
+
+# In[21]:
+
+
+import numpy as np
+import math 
+import pandas as pd
+import os
+from os import path
+import sqlite3
```

Comment:
Unless I'm overlooking something, we're only using pandas in this notebook right? If so, it is safe to delete the other packages.